smart_open/doctools.py

```diff
@@ -13,6 +13,8 @@
 
 import inspect
 import io
+import os.path as P
```

Comment:
-1 on import aliases. Please use the full name – especially with built-ins.

Reply:
OK, done